.tekton/listeners/pr.yaml

```diff
@@ -37,6 +37,10 @@ spec:
       value: "default-pipeline"
     - name: skip-cache
       value: $(params.skip-cache)
+    - name: prerelease
```

Comment:
Double check if we need to include this in listeners other than `node-pre-release`.